### ISSUE TITLE
Cannot access the colour picker generated by `form.colour` via the keyboard

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1170]
Investigate if this is an issue - does the fact we provide a text input for the hex value negate this failure?

**Comment by Stanton** [comment]
There are no nice accessible colour pickers unless they're very limited in terms of the colour options available. I'm of the opinion that because we provide the text input, the control provides an accessible method of use.
